User/jw msft/rmv do ver

jw-msft opened the pull request.

Nox-MSFT reviewed (commented):
@jw-msft , Do we still need these code in adu_core_interface/cmakelists.txt?


if (ADUC_PLATFORM_LAYER STREQUAL "linux")
    find_package (deliveryoptimization_sdk CONFIG REQUIRED)
    target_link_libraries (${PROJECT_NAME} PRIVATE Microsoft::deliveryoptimization)
else ()
    message (FATAL_ERROR "Invalid platform layer specified: ${ADUC_PLATFORM_LAYER}")
endif ()

jw-msft commented:
> @jw-msft , Do we still need these code in adu_core_interface/cmakelists.txt?
> 
> if (ADUC_PLATFORM_LAYER STREQUAL "linux") find_package (deliveryoptimization_sdk CONFIG REQUIRED) target_link_libraries (${PROJECT_NAME} PRIVATE Microsoft::deliveryoptimization) else () message (FATAL_ERROR "Invalid platform layer specified: ${ADUC_PLATFORM_LAYER}") endif ()

Fixed.  Also removed unused cmake function in agentRules.cmake

jw-msft commented:
> @jw-msft , Do we still need these code in adu_core_interface/cmakelists.txt?
> 
> if (ADUC_PLATFORM_LAYER STREQUAL "linux") find_package (deliveryoptimization_sdk CONFIG REQUIRED) target_link_libraries (${PROJECT_NAME} PRIVATE Microsoft::deliveryoptimization) else () message (FATAL_ERROR "Invalid platform layer specified: ${ADUC_PLATFORM_LAYER}") endif ()

Fixed.  Also removed unused cmake function in agentRules.cmake
